.vib/appsmith/vib-publish.json

```diff
@@ -0,0 +1,85 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/appsmith"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/appsmith"
+        },
+        "runtime_parameters": "YmFja2VuZDoKICBhZG1pblBhc3N3b3JkOiAiYml0bmFtaSExMjM0IgogIGNvbnRhaW5lclBvcnRzOgogICAgaHR0cDogODgyMwogIHNlcnZpY2U6CiAgICBwb3J0czoKICAgICAgaHR0cDogODA4MwoKCnJ0czoKICBjb250YWluZXJQb3J0czoKICAgIGh0dHA6IDg4NTAKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDgwODUKY2xpZW50OgogIHNlcnZpY2U6CiAgICB0eXBlOiBMb2FkQmFsYW5jZXIKICAgIHBvcnRzOgogICAgICBodHRwOiA4MAogIGNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICAgIHJ1bkFzVXNlcjogMTAwMgogIHBvZFNlY3VyaXR5Q29udGV4dDoKICAgIGZzR3JvdXA6IDEwMDIKCg==",
```

Comment:
You are changing the values of `backend/rts.containerPorts` in your runtime_params but they can actually be tested from inside the client pod you are running goss in. I would drop them from here. On the other hand, you are not changing the value of `client.containerPorts.http`, which could be tested checking the `/opt/bitnami/appsmith/conf/docker.env` file.

Additionally, it would be good to include the minimum sub-charts's params for the chart to function properly and modify the default values that can be easily tested (via `docker.env`). It could be something similar to:

```
redis:
  enabled: true
  architecture: standalone
  master:
    service:
      ports:
        redis: 6380
mongodb:
  enabled: true
  architecture: replicaset
  replicaCount: 2
  auth:
    usernames:
      - bn_vib_appsmith
    databases:
      - bitnami_vib_appsmith
  service:
    ports:
      mongodb: 27018
  arbiter:
    enabled: false
```